playground.sh

```diff
@@ -35,6 +35,9 @@ testDocker() {
     echo "ERROR: There was an issue running the hello-world container. Please check your Docker installation."
     exit 1
   fi
+  for containerId in $(docker ps -a | grep hello-world | awk '{print $1}'); do
+    docker rm $containerId
```

Comment:
Please add `docker rmi hello-world:latest`  here.

and modify `docker run --pull always hello-world >/dev/null 2>&1` to `docker run --pull always hello-world:latest >/dev/null 2>&1`

Reply:
@xunliu Add loop to remove images because sometimes there's two tags in local.
```
 docker images | grep hello-world
hello-world                                    latest                                    ee301c921b8a   20 months ago   9.14kB
hello-world                                    linux                                     ee301c921b8a   20 months ago   9.14kB

```